docs/conf.py

```diff
@@ -187,6 +180,7 @@ def setup(app):
 
 
 # -- Extension configuration -------------------------------------------------
+bibtex_bibfiles = ['..\morty\external\literature.bib']
```

Comment:
the previous literature.bib file pointed back at this location, this seems simpler